hvpy/core.py

```diff
@@ -0,0 +1,47 @@
+import requests
+
+from hvpy.io import HvpyParameters, OutputType
+
+
+def parse_response(response: requests.Response, output_parameters: OutputType):
+    """
+    Parses the response from the API call based on the output type.
+
+    Parameters
+    ----------
+    response : requests.Response
+        The response from the API call.
+    output_parameters : OutputType
+        The output type.
+
+    Returns
+    -------
+    parsed_response : binary, str or json
```

Comment:
```suggestion
    binary | str | json
```
I think that's right.